src/components/user-defined-sets/SetsManager.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import CurrentSetManager from './CurrentSetManager';
+
+export default function SetsManager(props) {
+  const {
+    sets,
+    onUpdateSets = (msg) => {
+      console.warn(`onUpdateSets from SetsManager ${msg}`);
+    },
+  } = props;
+  return (
+    <div className="sets-manager">
+      <CurrentSetManager sets={sets} onUpdateSets={onUpdateSets} />
+      <div className="named-set-list">
+        {Array.from(sets.namedSets.keys()).map(key => (
```

Comment:
Do we have a guarantee that the key order is stable? I wonder if the result should be sorted...

Reply:
Right now `namedSets` is an Immutable OrderedMap so it should be stable https://immutable-js.github.io/immutable-js/docs/#/OrderedMap but we could also use a regular map and sort based on some criteria.